test/sequential/test-fs-watchfile.js

```diff
@@ -64,23 +64,47 @@ fs.watchFile(enoentFile, {interval: 0}, common.mustCall(function(curr, prev) {
   }
 }, 2));
 
-// Watch events should callback with a filename on supported systems
-if (common.isLinux || common.isOSX || common.isWindows || common.isAix) {
-  const dir = common.tmpDir + '/watch';
 
-  fs.mkdir(dir, common.mustCall(function(err) {
-    if (err) assert.fail(err);
-
-    fs.watch(dir, common.mustCall(function(eventType, filename) {
-      clearInterval(interval);
-      this._handle.close();
-      assert.strictEqual(filename, 'foo.txt');
-    }));
+function WatchTestCase(is, dirName, fileName, field) {
```

Comment:
`is` is a very nondescriptive name… I’d probably use the reverse and call it `skip`.